src/components/AddressInfo.js

```diff
@@ -4,11 +4,13 @@
 class AddressInfo {
     /**
      * @param {{ userFriendlyAddress: string, label: string?, imageUrl: URL?, accountLabel: string?}} addressInfo
+     * @param {boolean} [displayAsCashlink = false]
      */
-    constructor(addressInfo) {
+    constructor(addressInfo, displayAsCashlink = false) {
```

Comment:
The `displayAsCashlink` could also be be part of the `addressInfo`, maybe as `isCashlink`.